torch/testing/_internal/distributed/_tensor/common_dtensor.py

```diff
@@ -69,13 +69,14 @@ def reset_parameters(self):
 @dataclass
 class ModelArgs:
     n_layers: int = 2
-    vocab_size: int = 16
-    max_seq_len: int = 16
-    dim: int = 8
+    vocab_size: int = 28
+    max_seq_len: int = 32
```

Comment:
curious why we need to enlarge the sizes here?

Reply:
These new parameters didn't work before we use double precision. Since we know they will work, I just changed back to old parameters to minimize change.